CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Better handle deinversion of special roles ([#10][])
 * Get `smatch-table.py` working again (part of PR [#27][])
 * Add `tests/` subdirectory and `test_top.py` ([#25])
+* Make TOP relation an attribute of the top variable ([#25])
```

Comment:
Make TOP relation's value constant?

Reply:
You're right, it was already an attribute. And I guess it was already a "constant" in that it was not a variable. I'll clarify that it's no longer the top node's concept.